pkg/network/vmispec/interface.go

```diff
@@ -188,3 +188,22 @@ func HasBindingPluginDeviceInfo(iface v1.Interface, bindingPlugins map[string]v1
 	}
 	return false
 }
+
+func IndexPodIfacesNamesByNetworkName(networks []v1.Network, interfaceStatus []v1.VirtualMachineInstanceNetworkInterface) (map[string]string, error) {
+	podIfacesByNetName := map[string]string{}
+
+	for _, network := range networks {
+		is := LookupInterfaceStatusByName(interfaceStatus, network.Name)
```

Comment:
You can use `i` or `iface` or `ifaceStatus`. `is` is hard for me to process.

Reply:
Done.